mkvdts2ac3.sh

```diff
@@ -367,7 +367,7 @@ else
 fi
 
 # Added check to see if AC3 track exists. If so, no need to continue
-if [ "$(mkvmerge -i "$MKVFILE" | grep -i "${AUDIOTRACKPREFIX}AC3")" ]; then
+if [ "$(mkvmerge -i "$MKVFILE" | grep -i "${AUDIOTRACKPREFIX}AC-3")" ]; then
```

Comment:
Should we support both?

Reply:
From what I can see, this change was made in 2015, so a long time ago:
https://gitlab.com/mbunkus/mkvtoolnix/commit/f1b5f1bb1669f4f88eb40c8b8dbd550d8aeaee91

Feels pretty safe to just take this change as is since this changed in v8.5.0 and we are now on v35.x.